client/rhel/spacewalk-client-tools/src/up2date_client/rhnreg_constants.py

```diff
@@ -19,77 +19,76 @@
 COPYRIGHT_TEXT        = _(u"Copyright © 2006--2014 Red Hat, Inc. All rights reserved.")
 
 # Satellite URL Window
-SATELLITE_URL_WINDOW  = _("Enter your Red Hat Satellite URL.")
-SATELLITE_URL_TEXT    = _("Please enter the location of your Red Hat Satellite "
+SATELLITE_URL_WINDOW  = _("Enter your Uyuni URL.")
+SATELLITE_URL_TEXT    = _("Please enter the location of your Uyuni "
                           "server and of its SSL "
                           "certificate. The SSL certificate is only required "
                           "if you will be connecting over https (recommended).")
-SATELLITE_URL_PROMPT  = _("Satellite URL:")
+SATELLITE_URL_PROMPT  = _("Uyuni URL:")
 SATELLITE_URL_PROMPT2 = _("SSL certificate:")
-SATELLITE_REQUIRED    = _("You must enter a valid Satellite URL.")
+SATELLITE_REQUIRED    = _("You must enter a valid Uyuni URL.")
 SSL_REQUIRED          = _("If you are using https you must enter the location "
                           "of a valid SSL certificate.")
 
 # Connect Window
 CONNECT_WINDOW        = _("Attempting to contact the Spacewalk server.")
-CONNECT_WINDOW_TEXT   = _("We are attempting to contact the Red Hat "
-                          "Network server at %s.")
+CONNECT_WINDOW_TEXT   = _("We are attempting to contact the Uyuni "
+                          "server at %s.")
 CONNECT_WINDOW_TEXT2  = _("A proxy was specified at %s.")
 
 # Start Window
 START_REGISTER_WINDOW = _("System Registration")
 START_REGISTER_TEXT   = _("This assistant will guide you through "
                           "connecting your system to "
-                          "Red Hat Satellite to receive software "
+                          "Uyuni to receive software "
```

Comment:
```suggestion
                          "Uyuni to receive software "
```

Definitely need a PRODUCT_NAME replacement to make this unsurprising for downstream users :-(

Reply:
Done.